app/javascript/components/supplemental_application/sections/LeaseInformationInputs.js

```diff
@@ -37,46 +38,72 @@ const toggleNoPreferenceUsed = (formApi, value) => {
-          field='lease_start_date'
-          label='Lease Start Date'
-          type='date'
-          pattern='[0-9]{4}-[0-9]{2}-[0-9]{2}'
-          placeholder='YYYY-MM-DD'
-        />
-        <CustomFormGrid.Select
-          label='Preference Used'
-          onChange={(value) => toggleNoPreferenceUsed(formApi, value)}
-          field='preference_used' options={confirmedPreferenceOptions} placeholder='Select One' />
-      </FormGrid.Row>
-      <FormGrid.Row paddingBottom>
-        <CustomFormGrid.Text
-          label='Monthly Unit Rent'
-          field='total_monthly_rent_without_parking' type='number' placeholder='Enter Amount' />
-        <CustomFormGrid.Text
-          label='Monthly Parking Rent'
-          field='monthly_parking_rent' type='number' placeholder='Enter Amount' />
-        <CustomFormGrid.Plain
-          label='Total Monthly Rent'
-          text={String(totalMonthlyRent)} />
-        <CustomFormGrid.Text
-          label='Monthly Tenant Contribution'
-          field='monthly_tenant_contribution' type='number' placeholder='Enter Amount' />
-      </FormGrid.Row>
-    </React.Fragment>
+    <NestedForm field='lease' >
+      <Form defaultValues={formApi.values['lease']} validateError={validateError}>
+        { nestedFormApi => {
+          const totalMonthlyRent = getTotalMonthlyRent(nestedFormApi.values)
+          return (
+            <React.Fragment>
+              <FormGrid.Row paddingBottom>
+                <CustomFormGrid.Select
+                  label='Assigned Unit Number'
+                  field='unit' options={availableUnitsOptions} placeholder='Select One' />
+                <CustomFormGrid.Text
+                  field='lease_start_date'
+                  label='Lease Start Date'
+                  type='date'
+                  pattern='[0-9]{4}-[0-9]{2}-[0-9]{2}'
+                  placeholder='YYYY-MM-DD'
+                />
+                <CustomFormGrid.Select
+                  label='Preference Used'
+                  onChange={(value) => toggleNoPreferenceUsed(nestedFormApi, value)}
+                  field='preference_used' options={confirmedPreferenceOptions} placeholder='Select One' />
+              </FormGrid.Row>
+              <FormGrid.Row paddingBottom>
+                <FormGrid.Item>
+                  <Field.Text
+                    label='Monthly Unit Rent'
+                    field='total_monthly_rent_without_parking'
+                    placeholder='Enter Amount'
+                    errorMessage={(label, error) => error} />
+                </FormGrid.Item>
+                <FormGrid.Item>
+                  <Field.Text
+                    label='Monthly Parking Rent'
+                    field='monthly_parking_rent'
+                    placeholder='Enter Amount'
+                    errorMessage={(label, error) => error} />
+                </FormGrid.Item>
+                <CustomFormGrid.Plain
+                  label='Total Monthly Rent'
```

Comment:
this has issues with the currency fields, but will be removed in https://www.pivotaltracker.com/story/show/163567761 so I'm not going to fix it here